project/plugins.sbt

```diff
@@ -9,6 +9,10 @@ resolvers ++= Seq(
   Classpaths.sbtPluginReleases
 )
 
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "0.5.0")
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.3")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
+
+addSbtPlugin("org.ensime" % "sbt-ensime" % "2.5.1")
```

Comment:
Why are you adding this to the project. You can set this in your machine globals. This brings a lot of dependencies and other things that will have unneeded conflicts.